src/index.html

```diff
@@ -1,12 +1,388 @@
+<!-- #region head -->
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <!-- #endregion -->
+  <!-- #region header -->
+  <script type="text/javascript" src="./scripts/main.js" ></script>
+  <header class="header">
+    <a href="#home" class="header__logo">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+        class="header__logo-img"
+      >
+    </a>
+    <nav class="nav">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
    </a>
    
    <nav class="nav">
```